app/controllers/account/profile.js

```diff
@@ -1,4 +1,17 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
+  actions: {
+    async updateContactInfo() {
+      this.set('isLoading', true);
+      try {
+        await this.model.save();
+        this.notify.success(this.l10n.t('Your Contact Info has been updated'));
+      }  catch (error) {
+        this.notify.error(this.l10n.t(error.message));
+      } finally {
```

Comment:
Finally is not needed here at all. `this.set('isLoading', false);` can be out of the try-catch block and it'll be executed anyway either on success or fail.

Reply:
@niranjan94 Updated. Please have a look